src/main/java/com/hubspot/jinjava/lib/filter/TruncateHtmlFilter.java

```diff
@@ -55,6 +56,47 @@ public String getName() {
     return "truncatehtml";
   }
 
+  @Override
+  public Object filter(
+    Object var,
+    JinjavaInterpreter interpreter,
+    Object[] args,
+    Map<String, Object> kwargs
+  ) {
+    String length = null;
+    if (kwargs.containsKey(("length"))) {
+      length = Objects.toString(kwargs.get("length"));
+    }
+    String end = null;
+    if (kwargs.containsKey("end")) {
```

Comment:
Changed this to use kwargs - there's probably a more generic way of doing this, but I figure that can be tackled when trying to bring this functionality to our other filters.